<ui version="4.0" >
 <class>pqPluginDialog</class>
 <widget class="QDialog" name="pqPluginDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Plugin Manager</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" colspan="2" >
    <widget class="QLabel" name="HelpText" >
     <property name="text" >
      <string>TextLabel</string>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="remoteGroup" >
     <property name="title" >
      <string>Remote Plugins</string>
     </property>
     <layout class="QGridLayout" >
      <item row="1" column="0" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1" >
       <widget class="QPushButton" name="loadRemote" >
        <property name="text" >
         <string>Load</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2" >
       <widget class="QListWidget" name="remotePlugins" >
        <property name="textElideMode" >
         <enum>Qt::ElideMiddle</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QGroupBox" name="localGroup" >
     <property name="title" >
      <string>Local Plugins</string>
     </property>
     <layout class="QGridLayout" >
      <item row="1" column="1" >
       <widget class="QPushButton" name="loadLocal" >
        <property name="text" >
         <string>Load</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0" colspan="2" >
       <widget class="QListWidget" name="localPlugins" >
        <property name="textElideMode" >
         <enum>Qt::ElideMiddle</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QComboBox" name="RecentRemoteCombo" >
     <item>
      <property name="text" >
       <string>Recent Remote Plugins...</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QComboBox" name="RecentLocalCombo" >
     <item>
      <property name="text" >
       <string>Recent Local Plugins...</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0" colspan="2" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>pqPluginDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>pqPluginDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
